blog/PyMC_Past_Present_Future.md

```diff
@@ -0,0 +1,168 @@
+
+This presented a challenge for PyMC, which relied heavily on Theano. The PyMC team spent over a year evaluating other computational backends, including MXNet, TensorFlow, and PyTorch, before deciding to try TensorFlow.  TensorFlow had been supportive of PyMC and was developing TensorFlow Probability, which included components that would be useful for PyMC. However, the rapid changes to TensorFlow presented many challenges.
+
+![Theano with JAX](../_static/chris-pymcon-keynote/Theano_with_Jax.png)
+
+In 2020, while working on the symbolic PyMC project, Brandon Willard had the idea to link Theano to [JAX](https://jax.readthedocs.io/en/latest/notebooks/quickstart.html) as a computational back end. This would allow Theano to take advantage of JAX's autograd and linear algebra acceleration capabilities without the constraints of a deep learning framework. Willard successfully developed a JAX linker for Theano and used it to create the Theano-PyMC library (later renamed to Aesara), to be used as a back-end for PyMC.
+
+The first [PyMCon](https://pymc-devs.github.io/pymcon/speakers) took place in October 2020. Chris Fonnesbeck closes out the talk by thanking everyone who worked to make the conference a reality, including Executive Directors Thomas Wiecki and Ravin Kumar. You can view the presentations here, or view the full talks on [the PyMCon playlist on YouTube](https://www.youtube.com/watch?v=UznM_-_760Y&list=PLD1x-BW9UdeHN2vwR6kIApJATd2jZzeya).
+
+## 2020
+
+![Golden Age](../_static/chris-pymcon-keynote/Present_day_golden_age.png)
+
+How far have we come from the early 2000s?
+
+As Chris Fonnesbeck says, we are currently in the “golden age of probabilistic programming” — there are  now many options to implement Bayesian models on different platforms.
+
+Today, when you build a model in PyMC, the interface is designed to be intuitive and easy to use. PyMC aims to make it as easy to code your model as it is to write it down on a whiteboard:
+
+![Model in PyMC](../_static/chris-pymcon-keynote/PyMC_model_sample_code.png)
+
+You can simply run the {func}`pymc.sample` function to fit your model, which automates many of the decisions involved in the process. This allows you to generate MCMC results with just one line of code, without having to worry about the technical details of the algorithm. Thomas Wiecki, one of the founding developers, refers to this as the "automatic inference button".
+Chris's presentation was in October 2020.  Below find updates to the PyMC library since that keynote. 
+
+## 2022 (June)
+
+[PyMC v4.0 was released](https://www.pymc.io/blog/v4_announcement.html) in June 2020, which incorporated these major changes:  
+- PyMC3 was renamed PyMC
+- PyMC backend now used Theano; this new backend was renamed Aesara
+- JAX backend for faster sampling
+- Dynamic shape support
+- New website design:  [www.pymc.io/](https://www.pymc.io/.)
+
+## 2023 (November)
```

Comment:
Should We have a section mentioning PyMC-based software, I am mostly thinking of Bambi, which can be very friendly for newcomers, but we could mention other packages as well.